packages/create-redwood-app/template/web/src/pages/FatalErrorPage/FatalErrorPage.tsx

```diff
@@ -7,7 +7,13 @@
 // still render a generic error page, but your users will prefer something a bit more
 // thoughtful. =)
 
-export default () => (
+// Ensures that production builds do not include the error page
+let RedwoodDevFatalErrorPage = undefined
+if (process.env.NODE_ENV === 'development') {
+  RedwoodDevFatalErrorPage = require("@redwoodjs/web").DevFatalErrorPage;
+}
```

Comment:
Does this actually make sure it's not in the bundled app though? I don't know.

Reply:
Seem to be working!